modules/exploits/multi/http/struts_code_exec_classloader.rb

```diff
@@ -265,11 +271,8 @@ def class_loader_exploit
       fail_with(Failure::Unknown, "#{peer} - The log file hasn't been flushed")
     end
 
-    if target['Platform'] == 'win' && target['Arch'] == ARCH_X86
-      register_files_for_cleanup("../webapps/ROOT/#{@jsp_file}")
-    else
-      register_files_for_cleanup(@jsp_file)
-    end
+    # This path depends on CWD. May require manual cleanup
+    register_files_for_cleanup("webapps/ROOT/#{@jsp_file}")
```

Comment:
What do you think about removing the auto-cleanup for the JSP? maybe we can put directly:

```ruby
print_warning("This exploit may require manual cleanup of '#{f}' on the target")
```

In this way at least the user will know that he has to manually clean the JSP.